jsonrpc/dispatcher.go

```diff
@@ -403,7 +403,7 @@ func (d *Dispatcher) getNextNonce(address types.Address, number BlockNumber) (ui
 func (d *Dispatcher) decodeTxn(arg *txnArgs) (*types.Transaction, error) {
 	// set default values
 	if arg.From == nil {
-		return nil, fmt.Errorf("from is empty")
+		arg.From = &types.Address{}
```

Comment:
`&types.ZeroAddress` is more clear?

Reply:
👍 https://github.com/0xPolygon/polygon-sdk/pull/117/commits/11b0825ee878b1ca0ce3337ff79356e54d87d450